api/src/wfl/log.clj

```diff
@@ -0,0 +1,72 @@
+(ns wfl.log
+  "Logging for WFL"
+  (:require [clojure.core]
+            [clojure.data.json :refer [write-str]])
+  (:import [java.sql Timestamp]
+           [java.time OffsetDateTime]))
+
+(defn googleize-field
+  "Prepends the google logging domain to the fields that require it for Google Logging."
+  [field]
+  (let [field-name (name field)]
+    (if (some #(= field-name %) '("labels" "insertId" "operation" "sourceLocation" "spanId" "trace")) (str "logging.googleapis.com/" (name field-name))
+        field-name)))
+
+(defn write-to-stdout
```

Comment:
Any of these functions you don't expect clients to use, can you annotate them with `^:private`?